cli/dcoscli/tables.py

```diff
@@ -37,6 +37,7 @@ def task_table(tasks):
     fields = OrderedDict([
         ("NAME", lambda t: t["name"]),
         ("HOST", lambda t: t.slave()["hostname"]),
+        ("MESOS ID", lambda t: t["slave_id"]),
         ("USER", lambda t: t.user()),
         ("STATE", lambda t: t["state"].split("_")[-1][0]),
         ("ID", lambda t: t["id"]),
```

Comment:
maybe we should put this at the end, in case people have already done scripts parsing this and are cutting on column number?